hail/python/hailtop/pipeline/backend.py

```diff
@@ -194,7 +194,14 @@ def __init__(self, billing_project):
     def close(self):
         self._batch_client.close()
 
-    def _run(self, pipeline, dry_run, verbose, delete_scratch_on_exit, wait=True, open=False):  # pylint: disable-msg=R0915
+    def _run(self,
+             pipeline,
+             dry_run,
+             verbose,
+             delete_scratch_on_exit,
+             wait=True,
+             open=False,
+             batch_submit_args=None):  # pylint: disable-msg=R0915
```

Comment:
My version gives me readable names not codes.  Can we use the readable names and not the codes?  I have no idea what's being disabled here.